docs/start/resourceconfig.md

```diff
@@ -0,0 +1,78 @@
+# Configure Resource Limits for Azure Pipelines Agent
+
+## Linux
+
+### Memory
+When the agent on a Linux system that is under high memory pressure, it is important to ensure that the agent does not get killed or become otherwise unusable.
```

Comment:
This is great intro material. It helps motivate what we're doing here; I'd move it up to the top (line 2-ish).